### PR TITLE
Cm/erb and layout

### PR DESCRIPTION
**Pull request by CheezItMan**

### CODE REVIEW

File: intro-to-rails/erb-and-layout.md [hunk 1]
```diff
@@ -219,6 +221,145 @@ If you examine the layout above you will notice embedded ruby with `stylesheet_l
...
+How does a rails view get data from the controller?
+
+##### !end-question
+
+##### !options
+
+* Through parameters
+* Through instance variables, example: `@books`
+* From the model directly
+* Carrier pidgeon
```
Comment (tildeee) [inline intro-to-rails/erb-and-layout.md:340]: 😆

File: intro-to-rails/erb-and-layout.md [hunk 2]
```diff
@@ -219,6 +221,145 @@ If you examine the layout above you will notice embedded ruby with `stylesheet_l
...
+* Through instance variables, example: `@books`
+
+##### !end-answer
+
+<!-- other optional sections -->
+<!-- !hint - !end-hint (markdown, users can see after a failed attempt) -->
+<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
+##### !explanation
+
+The controller will load data into instance variables and then the views will have access to that data.
```
Comment (tildeee) [inline intro-to-rails/erb-and-layout.md:355]: Awesome

File: intro-to-rails/erb-and-layout.md [hunk 3]
```diff
@@ -219,6 +221,145 @@ If you examine the layout above you will notice embedded ruby with `stylesheet_l
...
+
+* type: multiple-choice
+* id: c3f80eb9-2ca0-4b5b-993b-a6338e8c934f
+* title: How does information get sent from the controller to the view?
+* points: 1
+* topics: rails-views
+
+##### !question
+
+How does a rails view get data from the controller?
```
Comment (tildeee) [inline intro-to-rails/erb-and-layout.md:331]:
```suggestion
How does a Rails view get data from the controller?
```

File: intro-to-rails/erb-and-layout.md [hunk 4]
```diff
@@ -219,6 +221,145 @@ If you examine the layout above you will notice embedded ruby with `stylesheet_l
...
+
+Unlike the `<% %>` tag the `<%= %>` tag ___________.
+
+##### !end-question
+
+##### !options
+
+* Acts as a comment
+* Works in a CSS file
+* Places the return value of the ruby code into the HTML
```
Comment (tildeee) [inline intro-to-rails/erb-and-layout.md:295]:
```suggestion
* Places the return value of the Ruby code into the HTML
```

File: intro-to-rails/erb-and-layout.md [hunk 5]
```diff
@@ -219,6 +221,145 @@ If you examine the layout above you will notice embedded ruby with `stylesheet_l
...
+
+##### !end-options
+
+##### !answer
+
+* Places the return value of the ruby code into the HTML
```
Comment (tildeee) [inline intro-to-rails/erb-and-layout.md:301]:
```suggestion
* Places the return value of the Ruby code into the HTML
```

**Review by tildeee — APPROVED** [review]
The questions are awesome! I like how they're written!!

I didn't watch the video

If you take my suggestions on capitalizing Ruby and Rails, be sure to grab all of them, since one of them changes the "answer" to one of the MC questions.

**Comment by CheezItMan** [comment]
Thanks @tildeee !

**Pull request merged**